### ISSUE TITLE
Can a Service Fabric App be Integration Tested on Microsoft Hosted Agents?

### ISSUE THREAD

**Issue by JustinKaffenberger** [microsoft/service-fabric#923]
Our organization heavily utilizes Azure DevOps. Our Service Fabric applications are broken up into many Git repositories. While we have development clusters for multi-app integration testing, it would be nice to run some smoke tests directly in the build agent that is compiling the repository and running unit tests. Based on the [Microsoft Hosted Image](https://github.com/actions/virtual-environments/blob/master/images/win/Windows2019-Readme.md) we are using, it appears the Service Fabric SDK is installed. It also appears the `FabricHostSvc.exe` is indeed installed in the image. However, when I go to start up the local cluster using the `StartLocalCluster` function from `C:\Program Files\Microsoft SDKs\Service Fabric\Tools\Scripts\ClusterSetupUtilities.ps1`, it simply times out. Here is the output of that command:

```
Performing Start-Service on: FabricHostSvc . This may take a few minutes...
Could not perform the operation:Start-Service on the service:FabricHostSvc.
```

It would be great to know if there is anything I'm missing, or if this is even possible. It would greatly speed up validation of our applications in isolation, prior to promoting them to an actual release pipeline to a live cluster.

Much appreciated!

**Comment by craftyhouse** [comment]
@JustinKaffenberger Did you get assistance on this topic?

**Comment by JustinKaffenberger** [comment]
@craftyhouse no. I'm mainly curious because there are a variety of health checks we could do directly in the pipeline if the VM allowed a single node cluster to run. Obviously for large integration tests it will not work, but we would like to at least be able to do small sanity checks in the pipeline and fail our CI/CD process as early as possible when things are wrong with the Service Fabric application.
